Sources/SwiftJWT/JWT.swift

```diff
@@ -46,11 +46,11 @@ public struct JWT<T: Claims>: Codable {
     /// - Parameter header: A JSON Web Token header object.
     /// - Parameter claims: A JSON Web Token claims object.
     /// - Returns: A new instance of `JWT`.
-    public init(header: Header = Header(), claims: T) {
+    public init(header: H, claims: T) {
```

Comment:
This would be a breaking change.  Is it possible to retain the default value for `header` so that it is non-breaking? (ie. the tests should not have to change, unless you are adding one to test the new facility).